psm/CHANGELOG

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the [Haskell Package Versioning Policy](https://pvp.haskell.org/).
 
+## [0.5.2.1] - 2023-02-07
+
+### Removed
```

Comment:
Unnecessary documenting internals is an antipattern

Reply:
Fair enough, I removed it and added info about the change in downstream nix imports.